akka-http-core/src/main/scala/akka/http/impl/util/StageLoggingWithOverride.scala

```diff
@@ -1,9 +1,7 @@
-/**
- * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
- *
- * Copied and adapted from akka-remote
```

Comment:
Additional text is not carried over. Not sure if that would be a requirement.

Reply:
Would be nice if that could be prevented. The problem is that the commit which changes all the headers is so big that it's likely that changes like this would be lost forever. As they are likely copyright-related, maybe we shouldn't take chances there? Could we use a pattern like

```
"(?s).* Copyright \\([Cc]\\) (\\d{4}(-\\d{4})?) (Lightbend|Typesafe) Inc[^\n]*\n((?:\\s|\\*|\\*/)*)".r
```

to prevent those headers to be changed? Or to make sure that it doesn't touch the existing text?